glide.lock

```diff
@@ -0,0 +1,51 @@
+hash: db55a031aaa2f352fa5e9e4fda871039afb80e383a57fc77e4b35114d47cca8a
```

Comment:
Yeah, it's modeled after a bunch of projects, including CPAN, Gems, npm, Composer, and Mix. We did a pretty deep analysis of package managers. And I wouldn't really call it my brainchild. When I started it a few years ago, it barely resembled what it is now.

So it uses the official Go vendor/ recommendation, and caches dependencies in a vendor folder. The lock model is similar to godeps. But it solves some additional problems.